CyberHealth/users/views.py

```diff
@@ -10,17 +10,35 @@
 
 
 def send_user_notification(user_details, user_token, template_id='63d94931-3b5a-42dc-ba0d-06b40902298b'):
+    print(f'{settings.CLOUDFOUNDRY_SPACE} is the current environment value.')
+    if settings.CLOUDFOUNDRY_SPACE in [settings.DEFAULT_CLOUDFOUNDRY_SPACE, settings.DEFAULT_CLOUDFOUNDRY_SPACE_UAT]:
+        account_verification_link = f'https://cyberhealth-{settings.CLOUDFOUNDRY_SPACE}.london.cloudapps.digital/account_verification/{user_token}'
+    elif settings.CLOUDFOUNDRY_SPACE == settings.DEFAULT_CLOUDFOUNDRY_SPACE_FINAL:
+        account_verification_link = f'https://cyberhealth.london.cloudapps.digital/account_verification/{user_token}'
+    else:
+        account_verification_link = f'http://{settings.CLOUDFOUNDRY_SPACE}:8000/account/account_verification/{user_token}'
     return settings.NOTIFICATIONS_CLIENT.send_email_notification(
         email_address=user_details.email,
         template_id=template_id,
         personalisation={
             'first_name': user_details.first_name,
-            'account_verification':
-            f'http://127.0.0.1:8000/account_verification/{user_token}',
+            'account_verification': account_verification_link,
         }
     )
 
 
+# def send_user_notification(user_details, user_token, template_id='63d94931-3b5a-42dc-ba0d-06b40902298b'):
```

Comment:
Probably best just to delete this - we can always revert to previous versions :)